app-misc/zellij/zellij-0.40.1-r1.ebuild

```diff
@@ -536,10 +536,14 @@ LICENSE+="
 
 SLOT="0"
 KEYWORDS="~amd64"
-IUSE="doc"
+IUSE="+man"
 
+RDEPEND="
+	net-misc/curl
+"
```

Comment:
For fun, I experimented with building this and got a warning I was kind of expecting.

```
 * QA Notice: cargo built package contains possibly vendored code
 * 
 * crate libnghttp2-sys: sfparse.o nghttp2_buf.o nghttp2_callbacks.o nghttp2_debug.o nghttp2_extpri.o nghttp2_frame.o nghttp2_hd.o nghttp2_hd_huffman.o nghttp2_hd_huffman_data.o nghttp2_helper.o nghttp2_http.o nghttp2_map.o nghttp2_mem.o nghttp2_npn.o nghttp2_option.o nghttp2_outbound_item.o nghttp2_pq.o nghttp2_priority_spec.o nghttp2_queue.o nghttp2_rcbuf.o nghttp2_session.o nghttp2_stream.o nghttp2_submit.o nghttp2_version.o
```

... sigh. You can't actually use the system copy: https://github.com/alexcrichton/nghttp2-rs/pull/5

Reply:
Do I need something special to see that QA notice?

```
$ ldd /usr/bin/zellij
        linux-vdso.so.1 (0x00007705883d5000)
        libcurl.so.4 => /usr/lib64/libcurl.so.4 (0x00007705882ab000)
        libc.so.6 => /lib64/libc.so.6 (0x0000770586815000)
        libgcc_s.so.1 => /usr/lib/gcc/x86_64-pc-linux-gnu/14/libgcc_s.so.1 (0x0000770588277000)
        libm.so.6 => /lib64/libm.so.6 (0x0000770586730000)
        libcares.so.2 => /usr/lib64/libcares.so.2 (0x0000770588242000)
        libnghttp2.so.14 => /usr/lib64/libnghttp2.so.14 (0x0000770588211000)
        libssh2.so.1 => /usr/lib64/libssh2.so.1 (0x00007705866db000)
        libpsl.so.5 => /usr/lib64/libpsl.so.5 (0x00007705866c6000)
        libssl.so.3 => /usr/lib64/libssl.so.3 (0x00007705865c9000)
        libcrypto.so.3 => /usr/lib64/libcrypto.so.3 (0x0000770586000000)
        libzstd.so.1 => /usr/lib64/libzstd.so.1 (0x0000770585e43000)
        libz.so.1 => /usr/lib64/libz.so.1 (0x00007705865a9000)
        /lib64/ld-linux-x86-64.so.2 (0x00007705883d7000)
        libidn2.so.0 => /usr/lib64/libidn2.so.0 (0x0000770586585000)
        libunistring.so.5 => /usr/lib64/libunistring.so.5 (0x0000770585c83000
   ```

Reply:
It's a custom install-qa-check.d thing I wrote.
[/usr/local/lib/install-qa-check.d/50cargo-vendored-code](https://github.com/user-attachments/files/16091845/50cargo-vendored-code.txt)